modules/javafx.graphics/src/main/native-glass/gtk/glass_window.cpp

```diff
@@ -227,39 +350,33 @@ void WindowContextBase::process_destroy() {
     can_be_deleted = true;
 }
 
-void WindowContextBase::process_delete() {
+void WindowContext::process_delete() {
+    LOG0("process_delete\n");
     if (jwindow && isEnabled()) {
+        LOG0("jWindowNotifyClose\n");
         mainEnv->CallVoidMethod(jwindow, jWindowNotifyClose);
         CHECK_JNI_EXCEPTION(mainEnv)
     }
 }
 
-void WindowContextBase::process_expose(GdkEventExpose* event) {
+void WindowContext::notify_repaint() {
     if (jview) {
-        mainEnv->CallVoidMethod(jview, jViewNotifyRepaint, event->area.x, event->area.y, event->area.width, event->area.height);
+//        LOG0("jViewNotifyRepaint\n");
```

Comment:
I would either uncomment this log or remove it

Reply:
Done